### PR TITLE
Add title to feedback UI

### PR DESCRIPTION
**Pull request by bsudekum**
Default:

simulator screen shot oct 30 2017 at 10 56 05 am

While recording:
simulator screen shot oct 30 2017 at 10 56 12 am

/cc @ericrwolfe @v8c3 @frederoni @1ec5

### CODE REVIEW

**Comment by ericrwolfe** [comment]
Let me get you a better mock-up for this before you merge in.

**Comment by ericrwolfe** [comment]
@bsudekum something like image

**Comment by bsudekum** [comment]
@ericrwolfe updated:
simulator screen shot - iphone 7 - 2017-10-31 at 17 26 26

File: MapboxNavigation/UIView.swift [hunk 1]
```diff
@@ -39,7 +39,7 @@ class RippleLayer: CAReplicatorLayer {
         }
     }
     var rippleRadius: CGFloat = 100
-    var rippleColor: UIColor = UIColor.white
+    var rippleColor: UIColor = .red
```
Comment (1ec5) [inline MapboxNavigation/UIView.swift]: Should we make this color stylable? The developer has the option to make the background dark, right? Red against a dark background doesn’t work very well.

Comment (bsudekum) [reply MapboxNavigation/UIView.swift]: Yeah let's do this afterwards.

**Pull request merged**